earth2studio/models/px/dlesym.py

```diff
@@ -32,6 +33,7 @@
     OmegaConf = None
```

Comment:
I think you need to add `hydra-data` to the dependencies, modulus doesnt install this by default but seems an arch in the model uses the instantiate method

Reply:
uv add hydra-core --extra dlesym

Reply:
Interesting, yeah that's probably buried in the `Module.from_checkpoint` instantiation and the dep was covered by using the physicsnemo container. Will add